virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -68,6 +68,9 @@ This release adds new features and enhancements related to the following compone
 +
 As a cluster administrator, you can prevent users from enabling VM delete protection xref:../../virt/managing_vms/virt-enabling-disabling-vm-delete-protection.adoc#virt-removing-vm-delete-protection_virt-enabling-disabling-vm-delete-protection[by removing the option at the cluster level].
 
+//CNV-54747
+* You can now flatten the virtual machine (VM) definition file when creating a VM using instance types and preferences. For more information, see link:https://access.redhat.com/solutions/7107803[Using InstancetypeReferencePolicy to expand VirtualMachines] in the Red Hat Customer Portal.
```

Comment:
This is specifically about expanding live objects in the cluster and not a static definition file. It can also be applied during creation and to all existing VM objects within a cluster.

```suggestion
* You can now optionally expand virtual machines (VMs) when using instance types and preferences. For more information, see link:https://access.redhat.com/solutions/7107803[Using InstancetypeReferencePolicy to expand VirtualMachines] in the Red Hat Customer Portal.
```